zerver/migrations/0270_preregistrationuser_user_avatar_url.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.10 on 2020-03-09 13:41
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0269_gitlab_auth'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='preregistrationuser',
+            name='user_avatar_url',
+            field=models.CharField(max_length=900, null=True),
+        ),
+    ]
```

Comment:
All the migrations will have to be cleaned up - I think we only need this one?

Reply:
Yes. I just delete the other files?

Reply:
In this case that should work. In general if you have a mess in migrations, I think it'd be simplest to delete all of them and do `manage.py makemigrations` and Django will generate the migrations needed based on the models changes you've made.